packages/react-styled-core/src/Alert/styles.js

```diff
@@ -0,0 +1,39 @@
+import useColorMode from '../useColorMode';
+import useTheme from '../useTheme';
+
+const baseProps = {
+  display: 'flex',
+  alignItems: 'center',
+  position: 'relative',
+  overflow: 'hidden',
+  pl: 16,
+  pr: 16,
+  pt: 10,
+  pb: 10,
```

Comment:
Pin just revised the space for the top and bottom padding. You can use `1x`, `2x`, `3x` , `4x` (`4px`, `8px`, `12px`, `16px`)  to define the space for margin and padding:

```js
{
  pl: '4x',
  pr: '4x',
  pt: '2x',
  pb: '2x',
}
```